corehq/apps/users/models.py

```diff
@@ -800,6 +813,8 @@ class CouchUser(Document, DjangoUserMixin, IsMemberOfMixin, UnicodeMixIn, EulaMi
     assigned_location_ids = StringListProperty()
     has_built_app = BooleanProperty(default=False)
 
+    reporting_metadata = SchemaProperty(ReportingMetadata)
```

Comment:
will review closer later but was thinking about this last night - is there a reason this needs to be on the user model as opposed to a separate thing we can join with in elasticsearch? just thinking that saving all this extra data will make that model heavier (and therefore slower to access when loaded on every page). have also been wanting to clear all the computed stuff out of the domain model at some point...